src/utilities.jl

```diff
@@ -231,7 +313,6 @@ simplifications include:
   * no `TypeVar`s;
   * no types;
   * no keyword default values;
-  * `?` printed where `#unused#` arguments are found.
```

Comment:
This looks like an unrelated change? It would also apply to `SIGNATURES`? It would be good to discuss this in a separate PR. Although I am not sure if we want this:

```
julia> h(x, ::Int, z) = x ;

# master
julia> printmethod(Docs.Binding(Main, :h), h, first(methods(h)))
"h(x, ?, z)"

# vs this PR
julia> printmethod(Docs.Binding(Main, :h), h, first(methods(h)))
"h(x, , z)"
```

Reply:
Using underscores might be the least confusing syntax here since it's already valid syntax:

```
julia> h(x, _, z) = x
h (generic function with 1 method)
```

I recall `?` being added prior to the official `_` syntax being introduced.

Either way, separate PR seems appropriate.

Reply:
I changed it to `_` and added a test for the `h(x, _, z)` function. Resolved in f75bad9, 87df61e and 6d9f77a.

Reply:
Changing to underscores is a good idea and the change LGTM. But to keep unrelated changes in separate commits, could you open a separate PR for this? I think it's only 2 lines + the test to move to move over?

Reply:
Sounds good! I will do that.